app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/checkemail/CheckEmailScreen.kt

```diff
@@ -198,9 +209,136 @@ private fun CheckEmailContent(
     }
 }
 
+@Suppress("LongMethod")
+@Composable
+private fun CheckEmailLegacyContent(
+    email: String,
```

Comment:
Can I assume this was all wholesale copied from the old code?

Reply:
💯